index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <script src="https://kit.fontawesome.com/5e67ba493b.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="logo"><h1>Welcome</h1></div>
+        <div class="menu-burger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
```

Comment:
- [ ] Since this hamburger is a navigation for the mobile version, I would advise you use the nav tag for semantic satisfaction.